src/scripts/services/okta-service.ts

```diff
@@ -66,24 +81,22 @@ export const validateCredentials = (
  */
 export const pingOktaServer = (
   clientId: string,
-  organisationUrl: string
+  url: string
 ): TE.TaskEither<Error, 'Okta server is up and running.'> =>
   pipe(
-    TE.tryCatch(
-      async () => {
-        return await fetch(
-          `${organisationUrl}/oauth2/default/.well-known/oauth-authorization-server?client_id=${clientId}`
-        );
-      },
-      (error: unknown) => {
-        // eslint-disable-next-line functional/no-conditional-statement
-        if (error instanceof Error) {
-          return error;
-        }
-        return new Error('Failed to ping okta server.', {
-          cause: error,
-        });
-      }
+    parseUrl(url),
```

Comment:
**Issue:** this should happen while processing the command line arguments (and happen consistently for all commands that accept an org URL).

Reply:
added in commit: 221668d5d0b826386b4476d753022859713b8b0c